solr/core/src/test/org/apache/solr/search/TestThinCache.java

```diff
@@ -126,35 +176,4 @@ public void testInitCore() throws Exception {
     // for the other node-level cache, simply check that metrics are accessible
     assertEquals(0, nodeMetricsSnapshot.get("CACHE.nodeLevelCache/myNodeLevelCache.size"));
   }
-
-  @BeforeClass
```

Comment:
Please never put lifecycle methods at the bottom!

Reply:
Could we have patterns like that caught in precommit or lint checker?    These are the kinds of things that should be checked automatically for us.   not putting lifecycle methods at the bottom isn't an actual pattern that I knew about for example ;-).  We need more support in keeping formatting clean.